src/Symfony/Component/Cache/Adapter/AbstractAdapter.php

```diff
@@ -336,19 +335,9 @@ public function __destruct()
         }
     }
 
-    private function getId($key, $ns = false)
+    private function getId($key)
```

Comment:
Sorry for adding comments in a merged PR, but this was merged a bit quickly. I have some concerns about how `validateKey()` is used. It's a validator, but it's used like a getter or a normalizer.

Instead of this:

```php
private function getId($key)
{
    return $this->namespace.CacheItem::validateKey($key);
}
```

Why not using it like this?

```php
private function getId($key)
{
    CacheItem::validateKey($key);

    return $this->namespace.$key;
}
```